### PR TITLE
Changes - edits

### PR DESCRIPTION
**Pull request by Pilargit12**
- AlertManager - Steven's request to edit
- hosting tiers table - Shelly's request to edit
- Request quota increase edit - new page with original documentation

### CODE REVIEW

File: src/docs/automation-and-resiliency/request-quota-increase-for-openshift-project-set.md [hunk 1]
```diff
@@ -84,17 +86,133 @@ Follow this process:
...
+
+* A pop-up will appear to allow space to include all the information you have collected and gathered, this place only allows for text information if you need to submit photos or files please do so in the next step as it is necessary and part of the request process
+
+### Step 6: Email the Platform Services team and book a meeting with them
+
+The last step is to email the request to the Platform Services team at [PlatformServicesTeam@gov.bc.ca](mailto:PlatformServicesTeam@gov.bc.ca) and include the following information: 
+
+* Attach screenshot of Sysdig dashboard (if available)
+* Attach additional images or files that supplement the already submitted information in the previous step
+* Book a meeting with the Platform Services team
```
Comment (caggles) [inline src/docs/automation-and-resiliency/request-quota-increase-for-openshift-project-set.md:151]: We don't typically require a meeting for every quota request as a rule. I think we should remove this line - Billy will request a meeting if he needs to follow up in a discussion.

File: src/docs/automation-and-resiliency/request-quota-increase-for-openshift-project-set.md [hunk 2]
```diff
@@ -84,17 +86,133 @@ Follow this process:
...
+
+### Step 5: Submit edit request changes 
+
+The Platform Services team needs to know if your application is using the current quotas efficiently. 
+
+* Once all the information has been compiled from [**Step 4**](#step-4-gather-information-for-your-request) click on the `Submit edit request` button
+
+* A pop-up will appear to allow space to include all the information you have collected and gathered, this place only allows for text information if you need to submit photos or files please do so in the next step as it is necessary and part of the request process
+
+### Step 6: Email the Platform Services team and book a meeting with them
```
Comment (caggles) [inline src/docs/automation-and-resiliency/request-quota-increase-for-openshift-project-set.md:145]:
```suggestion
### Step 6: Email the Platform Services team
```

**Review by mtspn — APPROVED** [review]
Looks good, especially like the examples of the details needed to justify the quota increase requests.

File: src/docs/automation-and-resiliency/request-quota-increase-for-openshift-project-set.md [hunk 3]
```diff
@@ -11,66 +11,71 @@ page_purpose: Discusses what a user needs to do to get the quota for a specific
...
 
-If you have successfully onboarded to Sysdig, we provide a dedicated document outlining the specific metrics and evidence needed when applying for additional resources. This document serves as a guide to assist you in compiling the necessary data effectively. Make sure to gather comprehensive information on resource consumption, covering CPU and memory usage, along with any other relevant metrics specific to your application. For more details on the required metrics and evidence, check the [resource monitoring dashboards](../app-monitoring/resource-monitoring-dashboards.md) guide page.
+## Steps to request quota increase
 
- If you have any issues onboarding to Sysdig, contact the Platform Services team on the applicable [Rocket.Chat channel](https://chat.developer.gov.bc.ca/channel/devops-sysdig).
+### Step 1: Access the Platform Product Registry 
+Go to the [Platform Product Registry](https://registry.developer.gov.bc.ca/). Through dashboard of the product you wish to increase the quota.
```
Comment (Pilargit12) [inline src/docs/automation-and-resiliency/request-quota-increase-for-openshift-project-set.md:50]:
```suggestion
Access the [Platform Product Registry](https://registry.developer.gov.bc.ca/) and use the dashboard of your chosen product to adjust the quota.
```

File: src/docs/automation-and-resiliency/request-quota-increase-for-openshift-project-set.md [hunk 4]
```diff
@@ -11,66 +11,71 @@ page_purpose: Discusses what a user needs to do to get the quota for a specific
...
+Before initiating the request, make sure you have completed the following steps:
+
+1. **Evaluate resource usage:** Analyze your project's resource consumption to determine if a quota increase is necessary. Utilize tools like the OpenShift Dashboard or the command-line interface (CLI) for resource monitoring.
+
+Use the following information to help determine the level of resource the application needs:
+
+- [Resource management guidelines](https://github.com/BCDevOps/developer-experience/blob/master/docs/ResourceManagementGuidelines.md)
+- [Application resource tuning](../automation-and-resiliency/application-resource-tuning.md)
+
+2. **Familiarize yourself with Sysdig (optional):** Setting up Sysdig can provide comprehensive insights into your project's performance and resource utilization. However, it is not mandatory for requesting a quota increase and you can use a different system to provide the information the Platform Services Team requires to process your request increase.  If you are using it we provide more information in the section
```
Comment (ShellyXueHan) [inline src/docs/automation-and-resiliency/request-quota-increase-for-openshift-project-set.md:42]:
```suggestion
2. **Familiarize yourself with Sysdig (optional):** Setting up Sysdig can provide comprehensive insights into your project's performance and resource utilization. However, it is not mandatory for requesting a quota increase and you can use a different system to provide the information the Platform Services Team requires to process your request increase.  If you are using it we provide more information in the section [Onboarding to application monitoring with Sysdig](../app-monitoring/sysdig-monitor-onboarding.md).
```

File: src/docs/automation-and-resiliency/request-quota-increase-for-openshift-project-set.md [hunk 5]
```diff
@@ -11,66 +11,71 @@ page_purpose: Discusses what a user needs to do to get the quota for a specific
...
+## Steps to request quota increase
 
- If you have any issues onboarding to Sysdig, contact the Platform Services team on the applicable [Rocket.Chat channel](https://chat.developer.gov.bc.ca/channel/devops-sysdig).
+### Step 1: Access the Platform Product Registry 
+Access the [Platform Product Registry](https://registry.developer.gov.bc.ca/) and use the dashboard of your chosen product to adjust the quota.
 
-## Request a quota increase
-Before you request a quota increase, make sure that your project is using its resources efficiently. The Platform Services team wants to be very confident your project needs more quota before they grant an increase.
+### Step 2: Identify page and potential changes for each section
+The product page has different sections:
```
Comment (ShellyXueHan) [inline src/docs/automation-and-resiliency/request-quota-increase-for-openshift-project-set.md:53]: does registry support changes in multiple sections during resource request?

File: src/docs/automation-and-resiliency/request-quota-increase-for-openshift-project-set.md [hunk 6]
```diff
@@ -84,14 +89,154 @@ Follow this process:
...
+
+Please provide information about how much resources the important pods are using right now.  If you can, also let us know how their resource usage will change if we increase the quota. You don't need to give us details about every single pod or deployment in your namespace. Just focus on the ones that matter for why you're asking for this increase.
+
+ **Collect your application metrics:** Compare the current resource utilization and compare it with the quota. Specify your project's current resource quotas and usage.
+
+  - Current quota in the project set
+  - Total CPU, memory or storage currently used and any other resources relevant to your application's requirements.
+- Present the resource monitors within your namespace in order to understand  the level of resource consumption in your application and how efficiently it utilizes resources
+- Sysdig offers a pre-built dashboard template designed to visualize and analyze metrics in your OpenShift project. To simplify the process of requesting a quota increase, consider utilizing the Sysdig dashboard template. Instructions can be found in [here](../app-monitoring/sysdig-monitor-create-monitoring-dashboards.md)
```
Comment (ShellyXueHan) [inline src/docs/automation-and-resiliency/request-quota-increase-for-openshift-project-set.md:120]:
```suggestion
  - Current quota in the project set
  - Total CPU, memory or storage currently used and any other resources relevant to your application's requirements
  - Present the resource monitors within your namespace in order to understand  the level of resource consumption in your application and how efficiently it utilizes resources
  - Sysdig offers a pre-built dashboard template designed to visualize and analyze metrics in your OpenShift project. To simplify the process of requesting a quota increase, consider utilizing the Sysdig dashboard template. Instructions can be found in [here](../app-monitoring/sysdig-monitor-create-monitoring-dashboards.md)
```

File: src/docs/automation-and-resiliency/request-quota-increase-for-openshift-project-set.md (hunk 5 above)
Comment (Pilargit12) [reply src/docs/automation-and-resiliency/request-quota-increase-for-openshift-project-set.md:53]: Yeah when testing, one has also the possibility to changing product owner's and technical lead's info

**Review by Amritpal-Nijjar — APPROVED** [review]

**Review by caggles — APPROVED** [review]
lgtm!

File: src/docs/automation-and-resiliency/request-quota-increase-for-openshift-project-set.md [hunk 7]
```diff
@@ -11,66 +11,71 @@ page_purpose: Discusses what a user needs to do to get the quota for a specific
...
+3. **Get in touch with the Platform Services Team**
+This process suggests good communication from your team and the Platform Services Team particularly if you have questions prior your quota increase request, please reach out to them at [PlatformServicesTeam@gov.bc.ca](mailto:PlatformServicesTeam@gov.bc.ca) and enter  subject: Quota request with your inquiry.
 
-If you have successfully onboarded to Sysdig, we provide a dedicated document outlining the specific metrics and evidence needed when applying for additional resources. This document serves as a guide to assist you in compiling the necessary data effectively. Make sure to gather comprehensive information on resource consumption, covering CPU and memory usage, along with any other relevant metrics specific to your application. For more details on the required metrics and evidence, check the [resource monitoring dashboards](../app-monitoring/resource-monitoring-dashboards.md) guide page.
+## Steps to request quota increase
```
Comment (w8896699) [inline src/docs/automation-and-resiliency/request-quota-increase-for-openshift-project-set.md:48]:
```suggestion
As part of our ongoing efforts to optimize resource allocation across our OpenShift environments, we request teams seeking a quota increase to provide detailed justifications. This structured approach ensures that all requests are evaluated fairly and resources are used efficiently.
```

File: src/docs/automation-and-resiliency/request-quota-increase-for-openshift-project-set.md [hunk 8]
```diff
@@ -84,14 +89,154 @@ Follow this process:
...
+* Attach screenshot of Sysdig dashboard (if available)
+* Attach additional images or files that supplement the already submitted information in the previous step
+* If you have further questions, it is also recommended to book a meeting with the Platform Services Team
+
+## Essential resources to improve your quota increase request 
+
+### Examples of requests
+These examples mainly concentrate on particular pods or deployments. When you're asking for a quota increase, it's typically to support a specific deployment or group of deployments. Focus your request explanations on these - there's no need to spend time outlining the resource allocations for every single pod in your namespace, just provide the information that's relevant to the request.
+
+If you're unsure, it's better to include more information than less!
```
Comment (w8896699) [inline src/docs/automation-and-resiliency/request-quota-increase-for-openshift-project-set.md:161]:
```suggestion
If you're unsure, it's better to include more information than less, the Platform Team is here to help!
```

**Pull request merged**